vision/unit_tests/test_annotations.py

```diff
@@ -73,7 +73,7 @@ def test_from_pb(self):
         annotations = self._make_one().from_pb(image_response)
         self.assertEqual(annotations.labels, [])
         self.assertEqual(annotations.logos, [])
-        self.assertEqual(annotations.faces, ())
+        self.assertEqual(annotations.faces, [])
```

Comment:
It returns a list from the [list comprehension](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2922/files#diff-764c6fd4c7ba386b0fd21683de027e6cR153) now, instead of the signature default which is `()`.